weni_cli/clients/cli_client.py

```diff
@@ -99,13 +99,17 @@ def _streaming_request(
                 try:
                     error_data = response.json()
                     message = error_data.get("message", f"Request failed with status code {response.status_code}")
+                    if response.status_code == 401:
```

Comment:
why not put just 1 if before the try statement, to avoid having it twice in this function?